apps/pipelines/nodes/nodes.py

```diff
@@ -486,6 +486,8 @@ def _process_conditional(self, state: PipelineState, node_id=None):
 
         if not keyword:
             keyword = self.keywords[0]
+        else:
+            keyword = self._get_keyword(keyword)
```

Comment:
I don't think this is necessary since the pydantic validation will already have verified that the keyword is one from the list specified so we should just be able to return it as is.